src/components/content/Cards/Card.tsx

```diff
@@ -310,7 +310,13 @@ export const Card = ({
           )}
         </Box>
         <Box
-          sx={{ marginBottom: '20px' }}
+          sx={{
+            marginBottom: '20px',
+            overflow: 'hidden',
+            display: '-webkit-box',
```

Comment:
Have you checked if it's not useful to have a fallback for older browsers? As we are in shared components, I would make sure to have maximum compatibility, just in case.  
```
  <Box
    sx={{
      marginBottom: '20px',
      overflow: 'hidden',
      display: '-webkit-box',
      WebkitBoxOrient: 'vertical',
      WebkitLineClamp: '6',
      display: 'box', // Fallback for older browsers
      boxOrient: 'vertical', // Fallback for older browsers
      lineClamp: '6', // Fallback for older browsers
    }}
    className="cx-card__content--wrapper"
  >
```

Reply:
yes @ybidois  it is right fallback for the older browser for reference please check on these articles. 
https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-line-clamp
 https://css-tricks.com/line-clampin/ .
https://learnbricksbuilder.com/control-text-line-length-with-line-clamp/.